src/MOI_wrapper/results.jl

```diff
@@ -0,0 +1,44 @@
+# MathOptInterface results
+	
+function MOI.get(model::Optimizer, ::MOI.TerminationStatus)
+	return model.inner.status
+end
+	
+function MOI.get(model::Optimizer, ::MOI.ObjectiveValue)
+    if model.inner.status == MOI.OPTIMIZE_NOT_CALLED
+	    @error "optimizer not called"
+	end
+	return model.inner.objval
```

Comment:
There is some weird indentation in this file

Reply:
not reproducible on my machine with Visual Studio Code will check again.